docs/command-reference/server-management/hello.md

```diff
@@ -6,24 +6,13 @@ description: Handshake with Redis
 
 ## Syntax
 
-    HELLO [protover [AUTH username password] [SETNAME clientname]]
+    HELLO [protover]
 
 **Time complexity:** O(1)
 
-Switch to a different protocol, optionally authenticating and setting the
-connection's name, or provide a contextual client report.
+Switch to a different protocol or provide a contextual client report.
 
-Redis version 6 and above supports two protocols: the old protocol, RESP2, and
-a new one introduced with Redis 6, RESP3. RESP3 has certain advantages since
-when the connection is in this mode, Redis is able to reply with more semantical
-replies: for instance, `HGETALL` will return a *map type*, so a client library
-implementation no longer requires to know in advance to translate the array into
-a hash before returning it to the caller. For a full coverage of RESP3, please
-[check this repository](https://github.com/antirez/resp3).
-
-In Redis 6 connections start in RESP2 mode, so clients implementing RESP2 do
-not need to updated or changed. There are no short term plans to drop support for
-RESP2, although future version may default to RESP3.
+At the moment, the RESP3 protocol is not supported.
```

Comment:
HELLO currently makes little sense for switching protocols, maybe just remvoe the argument?